soccer/NewRefereeModule.cpp

```diff
@@ -283,6 +283,8 @@ void NewRefereeModule::updateGameState(bool blueTeam) {
             break;
         case Command::FORCE_START:
             _state.gameState.state = GameState::Playing;
+            _state.gameState.ourRestart = false;
+            _state.gameState.restart = GameState::None;
```

Comment:
@chachmu are there any other places where the restart variables need to be reset?